airflow/contrib/hooks/sqoop_hook.py

```diff
@@ -93,6 +93,7 @@ def Popen(self, cmd, **kwargs):
                               stdout=subprocess.PIPE,
                               stderr=subprocess.STDOUT,
                               **kwargs)
+        self.sp = sp
```

Comment:
Can we do `self.sp = subprocess.` directly and reference everything to `self.sp`? This makes it a bit simpler.

Reply:
Hey Fokko, thanks for the response. Will go ahead and do that. Agreed it keeps it simpler